blocks/ai-chat/package.json

```diff
@@ -26,6 +26,7 @@
     "serve": "block-scripts serve"
   },
   "dependencies": {
+    "@blockprotocol/design-system": "0.0.1",
```

Comment:
This is the old package name

Reply:
Fixed in 5f6a5d975715c02950a37f36febfe887eec892fd